Feature detection isn't entirely working here. It does not respect Rust's
`allowed-features` flag. It should probably use something like
`version_check::supports_feature` instead:
https://github.com/tkaitchuck/aHash/issues/117

diff --git a/build.rs b/build.rs
index 8102686..d561c98 100644
--- a/build.rs
+++ b/build.rs
@@ -4,7 +4,7 @@ use std::io::Write;
 fn main() {
     // Niche optimizations for `Borrowed*` and `Owned*` depend on `rustc_attrs`
     // which, outside of `std`, are only available on nightly.
-    use_feature_or_nothing("rustc_attrs");
+    // use_feature_or_nothing("rustc_attrs");
 
     // Rust 1.56 and earlier don't support panic in const fn.
     if has_panic_in_const_fn() {
